src/discord/staffcommands.py

```diff
@@ -273,17 +270,17 @@ async def complete_button(
 
 
 class CronSelect(discord.ui.Select):
-    def __init__(self, docs, bot: PiBot):
+    def __init__(self, docs: list[Cron], bot: PiBot):
         options = []
-        docs.sort(key=lambda d: d["time"])
+        docs.sort(key=lambda d: d.time)
         counts = {}
-        for doc in docs[:20]:
-            timeframe = (doc["time"] - discord.utils.utcnow()).days
+        for doc in docs[:20]:  # FIXME: Magic number
```

Comment:
Not too sure if this magic number should be `DISCORD_AUTOCOMPLETE_MAX_ENTRIES` or something different.